assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-title-label-block/block.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "woocommerce/mini-cart-title-label-block",
+	"version": "1.0.0",
+	"title": "Mini Cart Title Label",
+	"description": "Block that displays the 'Your cart' part of the Mini Cart Title block.",
+	"category": "woocommerce",
+	"supports": {
+		"align": false,
+		"html": false,
+		"multiple": false,
+		"reusable": false,
+		"inserter": true,
+		"lock": false,
+		"color": {
+			"text": true,
+			"background": true
+		},
+		"typography": {
+			"fontSize": true
+		},
+		"spacing": {
+			"padding": true
+		}
+	},
+	"attributes": {
+		"label": {
+			"type": "string"
+		},
+		"lock": {
+			"type": "object",
+			"default": {
+				"remove": true,
```

Comment:
Ok, I'm a bit confused, but for some reason I'm able to remove this block. :confused: Do you know what might be going on?

![imatge](https://user-images.githubusercontent.com/3616980/229742069-f5e63f83-a596-4e53-9aee-6334c5eaf8fd.png)

Reply:
I don't understand, the `lock` settings are working well for the `items counter`, but doing nothing for the `title label`:
```json
"lock": {
	"type": "object",
	"default": {
		"remove": true,
		"move": true
	}
}
even when I add those settings ☝️ the block can be removed and moved 🤷

Reply:
Yeah... that's super confusing. I was tinkering around and couldn't fix it either. I think that's maybe a GB bug or we are using the API incorrectly. :confused: 

For now, I wonder if we should simply not allow removing those blocks? I consider that would be better than an inconsistent experience. What do you think?

Reply:
Agree, I just pushed the change.